front-end-node/profiler/SaveOverrides.js

```diff
@@ -0,0 +1,26 @@
+/*jshint browser:true, nonew:false*/
+/*global WebInspector:true*/
+
+WebInspector.saveAs = function(blob,filename) {
+  var url = URL.createObjectURL(blob);
+  var link = document.createElementNS("http://www.w3.org/1999/xhtml", "a");
```

Comment:
Linter fails here. Run `npm test` to see the result or check in here [travisci](https://travis-ci.org/node-inspector/node-inspector/builds/59524079)
Don't worry about all tests here, but linter errors is in scope of this pr.